apps/judicial-system/backend/src/app/modules/case/e2e/case.spec.ts

```diff
@@ -332,6 +341,7 @@ describe('Case', () => {
       await Case.create(getCaseData()).then(async () => {
```

Comment:
Just want to point out that you are mixing `async` and `await` with `then`. I'm assuming that all instances of `then` could be `await`. Not asking for this to change just pointing it out in case you were unaware :)